.github/workflows/pull_request.yml

```diff
@@ -24,4 +29,32 @@ jobs:
       - name: Assemble
         run: ./gradlew assemble
       - name: Test
-        run: ./gradlew test
\ No newline at end of file
+        run: ./gradlew test
+      - name: Prepare Slack notification message
+        id: prepare_slack_message
+        run: |
+          LIB_PROP_FILE="lib.properties"
+          VERSION_NAME=`cat $LIB_PROP_FILE | grep VERSION_NAME | cut -d = -f 2`
+          SITE_URL=`grep SITE_URL < $LIB_PROP_FILE | cut -d = -f 2`
```

Comment:
i could not wrap my head around from which source are you greping the SITE_URL until after unfortunate couple of minutes I've noticed that you are redirecting input :D why do you use `cat` for version_name and this other way for site url?

Reply:
Just inconsistency, no special reason for it 😄 Changed it to `cat` in 67d85ab5bcde1b53380d93246c3b8d7e994833f7, notification is now sent after Bintray deploy and not after MR check which was used only for testing purposes.